README.md

```diff
@@ -10,8 +10,8 @@ Learning][].
 For in-depth information on the Graph Visualizer, see this tutorial:
 [TensorBoard: Graph Visualization][].
 
-[TensorBoard: Visualizing Learning]: https://www.tensorflow.org/get_started/summaries_and_tensorboard
-[TensorBoard: Graph Visualization]: https://www.tensorflow.org/get_started/graph_viz
+[TensorBoard: Visualizing Learning]: https://www.tensorflow.org/r1/summaries
```

Comment:
This link 404s for me. Should we point at the newer tutorials https://www.tensorflow.org/tensorboard/get_started?

Reply:
done.